recipes/msys/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "desres-msys" %}
+{% set version = "1.7.266" %}
+{% set sha1 = "a37806e5dbb76b5e97c5d6fc3cc4b03542e5999a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/DEShawResearch/msys/archive/{{ version }}.tar.gz
+    sha1: {{ sha1 }}
+  - url: http://www.inchi-trust.org/download/105/INCHI-1-SRC.zip
+    sha1: e3872a46d58cb321a98f4fd4b93a989fb6920b9c
+    folder: external/inchi/INCHI-1-SRC
+  - url: https://sourceforge.net/projects/lpsolve/files/lpsolve/5.5.2.5/lp_solve_5.5.2.5_source.tar.gz
+    folder: external/lpsolve/lp_solve_5.5
+    sha1: 232ce8edb073c7c9b6d9100abf77038d989c45c8
```

Comment:
Isn't there an `lpsolve55` package already that you can link against?

Reply:
Yep, done